components/nexus-orient/pom.xml

```diff
@@ -62,15 +62,9 @@
     </dependency>
 
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <optional>true</optional>
-    </dependency>
-
-    <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>jul-to-slf4j</artifactId>
-      <optional>true</optional>
+      <scope>test</scope>
     </dependency>
```

Comment:
Several of the test classes (e.g. `BackupRestoreTrial`) rely on `org.slf4j.bridge.SLF4JBridgeHandler` which appears to be supplied by `jul-to-slf4j`.